.github/workflows/test.yml

```
@@ -4,13 +4,12 @@ on: [push, pull_request]
 
 jobs:
   build:
-
     name: Build and test
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: ['22.3']
-        elixir: ['1.10.4']
+        otp: ['23.1']
```

We still want to test with Erlang 22.3.x as well:

```suggestion
        otp: ['22.3', '23.1']
```

---

Done